optaweb-employee-rostering/image.yaml

```diff
@@ -104,6 +104,7 @@ modules:
             version: "3.6"
           - name: jboss-kie-jdk-clean
             version: "11"
+          - name: jboss-kie-common
```

Comment:
is this module required?
IIRC this module only includes functions related to ocp k8s api.

Reply:
@spolti I added it since it is the "base" common module for all our kie images. You are correct, it's not used yet by employee rostering or controller, but thought it might be good to include here so that it will be already in place if we need to use anything in it (or added to it) in the future, and the overhead is really just sourcing one short bash script. However if you feel strongly that it shouldn't be added until we absolutely need it there, I can remove it. (Here and in the rhdm + rhpam controllers.)